pan/functions.pan

```diff
@@ -218,3 +218,19 @@ function full_hostname_from_object = {
     return (OBJECT);
 };
 
+
+@documentation{
+    desc = Function to deduplicate the content of a list, only the first occurence of each element in the list will be kept in the final list returned to the caller
+}
+function unique_list = {
```

Comment:
The aim is to deduplicate a list of strings / long when generating the profile of a host, to manage a new entitlement structure (to be submitted later). The ordering is not important for us, so if this can be done a better & faster way, I'm all for it!

Reply:
if you are up to it and if it is worth it, following doc sand example PR can show you what is needed: https://www.quattor.org/development/panc-internals-overview.html#adding-functionality (and example https://github.com/quattor/pan/pull/118).

but just to be clear, i have no issue with getting this merged :wink:

Reply:
Not sure it will be worth the trouble... Maybe if this starts getting used heavily and becomes the bottleneck, but not sure this will! It's true that in Java it would just be a matter of doing a collection copy from the list to a set, and returning it as a list.

Reply:
Plus you would have to wait for a new panc release before you could use it, which would slow things down.